codex_of_the_damned/templates/archetypes/contributions/creeping-sabotage.html

```diff
@@ -0,0 +1,140 @@
+{% extends "archetypes/_layout.html" %}
+{% import "archetypes/contributions/_contributors.html" as contributor with context %}
+{% import "archetypes/contributions/creeping-sabotage.json" as deck_json %}
+
+{% block meta %}
+<meta property="decklist" content='{{ deck_json | tojson }}' />
+{% endblock %}
+
+{% block archetype_name %}
+<h1><img src="{{ url_for('static', filename='img/icons/clan-nosferatu.svg') }}"> Creeping Sabotage</h1>
+
+{{ contributor.present( "Martin Cuchet", _("12/09/2020") ) }}
```

Comment:
Maybe use an international format for dates, like 2020-12-09. This is less confusing for everyone on the translation side.